### PR TITLE
[feat 3] size and style modules in timeline according to complexity

### PR DESCRIPTION
**Pull request by albertjvm**

### CODE REVIEW

**Comment by g2graman** [comment]
@albertjvm Did you mean to remove module-list-item.component?

https://github.com/bridge-school/content-dashboard/pull/9/commits/8699619ec5f71bd4b7ff7f4aa5ed5191bb0f7de1#diff-aa16206625cf2405e5a9e2d4e4b75877

**Comment by albertjvm** [comment]
@g2graman You moved module-list-item.component into it's own directory. So when I rebased there were 2, I removed the old one.

**Comment by g2graman** [comment]
Oops, @albertjvm you're right. My apologies!

File: src/scenes/home/timeline/timeline.component.tsx [hunk 1]
```diff
@@ -28,19 +32,39 @@ const timelineTarget: DropTargetSpec<Props> = {
...
     <div className="flex flex-auto flex-row h-100">
         <div className="flex flex-auto flex-column items-center justify-center">
-            <span className="ba b--moon-gray w-75 h4"/>
+            <div className="flex flex-row ba b--moon-gray w-100 mw-100 h3 overflow-x-scroll">
+                {
+                    timeline.map((module: ClassModule, index: number) => (
+                        <div 
+                            key={index} 
+                            className={`flex ba white ph2 bg-${complexityColors[module.complexity]} items-center`} 
+                            style={{flex: module.complexity}}
```
Comment (g2graman) [inline src/scenes/home/timeline/timeline.component.tsx]: 👍

**Comment by albertjvm** [comment]
@g2graman Yup, that all makes sense. I've added the default colour and moved that into a utility function.

**Comment by BrendanFDMoore** [comment]
@albertjvm LGTM, but can you please remove the `yarn.lock` file? The project is using `npm` with `package-lock.json` right now, and we don't want to have competing sources of truth.

**Comment by g2graman** [comment]
@BrendanFDMoore @albertjvm Oops, that's my fault https://github.com/bridge-school/content-dashboard/commit/fb8ce574a7bacc79253543c6cbdd97430be513d0

I didn't know what people would want to use

**Comment by albertjvm** [comment]
@BrendanFDMoore @g2graman removed

**Comment by ShaunLloyd** [comment]
Closes #3

**Pull request merged**